core/include/array/array_schema.h

```diff
@@ -685,7 +688,7 @@ class ArraySchema {
    *    - TILEDB_RLE
    *    - TILEDB_BZIP2
    */
-  std::vector<int> compression_;
+  std::vector<Compressor> compression_;
```

Comment:
I'll try to make this consistent.  I prefer `compressor_` because it describes what is actually being referenced.